content/en/blog/_posts/2022-10-03-current-state-2019-third-party-audit.md

```diff
@@ -0,0 +1,116 @@
+determined by Severity Level + Likelihood of a successful vulnerability
+exploit. These estimates and values in the table below are the authors' 
+personal opinion. An individual or end users' threat model may rate the
+benefits to fix a particular issue higher or lower.
+
+{{</note>}}
+
+| Title                                                                            | Issue                                                           | Time Commitment | Complexity | Benefit to Ecosystem |
+|----------------------------------------------------------------------------------|-----------------------------------------------------------------|-----------------|------------|----------------------|
+| Kubernetes does not facilitate certificate revocation                            | [#81111](https://github.com/kubernetes/kubernetes/issues/81111) | High            | High       | Medium               |
+| Use of InsecureSkipVerify and other TLS weaknesses                               | [#81119](https://github.com/kubernetes/kubernetes/issues/81119) | High            | High       | Medium               |
+| Kubectl can cause a local Out Of Memory error with a malicious Pod specification | [#81123](https://github.com/kubernetes/kubernetes/issues/81123) | Medium          | Medium     | Medium               |
+| Improper fetching of PIDs allows incorrect cgroup movement                       | [#81124](https://github.com/kubernetes/kubernetes/issues/81124) | Medium          | Medium     | Medium               |
+| kubelet liveness probes can be used to enumerate host network                    | [#81129](https://github.com/kubernetes/kubernetes/issues/81129) | High            | High       | Medium               |
+| TOCTOU when moving PID to manager's cgroup via kubelet                           | [#81113](https://github.com/kubernetes/kubernetes/issues/81113) | Medium          | Medium     | Low                  |
+| Log rotation is not atomic                                                       | [#81132](https://github.com/kubernetes/kubernetes/issues/81132) | Medium          | Medium     | Low                  |
+| Arbitrary file paths without bounding                                            | [#81133](https://github.com/kubernetes/kubernetes/issues/81133) | Medium          | Medium     | Low                  |
+| Services use questionable default functions                                      | [#81138](https://github.com/kubernetes/kubernetes/issues/81138) | Medium          | Medium     | Low                  |
+| Use standard formats everywhere                                                  | [#81140](https://github.com/kubernetes/kubernetes/issues/81140) | High            | High       | Very Low             |
+| Hardcoded use of insecure gRPC transport                                         | [#81142](https://github.com/kubernetes/kubernetes/issues/81142) | High            | High       | Very Low             |
```

Comment:
Do we want to include finding 37 as well? It's still open and in need of a fix but is tracked in a new issue now.

Reply:
Updated this and switched the call to action from kubelet -> API Server